src/server/scripts/Northrend/IcecrownCitadel/boss_lady_deathwhisper.cpp

```diff
@@ -375,31 +403,27 @@ class boss_lady_deathwhisper : public CreatureScript
                     case EVENT_SPELL_SHADOW_BOLT:
                         if (Unit* target = SelectTarget(SELECT_TARGET_RANDOM))
                             me->CastSpell(target, SPELL_SHADOW_BOLT, false);
-                        events.RepeatEvent(urand(5000, 8000));
+                        events.RepeatEvent(urand(2450, 3600));
```

Comment:
I am not sure about this timer, it feels like 2s 2s 2s 2s, stop, 2s, 2s... to me